lib/_tls_wrap.js

```diff
@@ -1433,9 +1434,10 @@ Server.prototype.addContext = function(servername, context) {
     throw new ERR_TLS_REQUIRED_SERVER_NAME();
   }
 
-  const re = new RegExp('^' + StringPrototypeReplace(
-    StringPrototypeReplace(servername, /([.^$+?\-\\[\]{}])/g, '\\$1'),
-    /\*/g, '[^.]*'
+  const re = new RegExp('^' + StringPrototypeReplaceAll(
```

Comment:
`String.prototype.replaceAll` is not available to primordials because it can be disabled by `--no-harmony-string-replaceall`